CRM/Contact/Import/Parser/Contact.php

```diff
@@ -1665,7 +1665,7 @@ private function formatProfileContactParams(
 
         $loc = CRM_Utils_Array::key($index, $locationType);
 
-        $blockName = $this->getLocationEntityForKey($fieldName);
+        $blockName = strtolower($this->getFieldEntity($fieldName));
```

Comment:
We have a function that's more reliable than `strtolower` because it adds underscores for entities like `financial_type`: `CRM_Core_DAO_AllCoreTables::convertEntityNameToLower()`